.github/pull_request_template.md

```diff
@@ -16,13 +16,17 @@ Please check all options that are relevant
 # How Has This Been Tested?  
 **Tested Configuration**:  
 - [ ] Ansible, Specify version(s):  
-- [ ] Specify Distro & version(s):  
-- [ ] Specify Package distro & version(s):  
-- [ ] Packages compiled from source & their version:  
+- Installation method:  
+    - [ ] Binary install from source, version:  
+    - [ ] OS ackage repository, distro and version:  
+    - [ ] Local package session, version:  
```

Comment:
```suggestion
    - [ ] Local package server, version:  
```

…am I off base? What’s a local package session? I was thinking like a custom package that was built internally and distributed using the customer’s own infrastructure.

Reply:
Or maybe that’s still covered by the custom package option. @gregscds ?

Reply:
Yeah I'm confused by this one as well. Remember this is a public repo so we have to make sure the things in this template would make sense to almost anyone coming in and opening in an issue.